bulk_query_gen/influxdb/influx_iot_common.go

```diff
@@ -101,3 +102,25 @@ func (d *InfluxIot) IotAggregateKeep(qi bulkQuerygen.Query) {
 	q := qi.(*bulkQuerygen.HTTPQuery)
 	d.getHttpQuery(humanLabel, interval.StartString(), query, q)
 }
+
+func (d *InfluxIot) LightLevelEightHours(qi bulkQuerygen.Query) {
+	interval := d.AllInterval.RandWindow(8 * time.Hour)
+
+	var query string
+	if d.language == InfluxQL {
+		query = fmt.Sprintf(`SELECT level FROM light_level_room WHERE time > '%s' AND time < '%s' GROUP BY *`, interval.StartString(), interval.EndString())
+	} else {
+		query = fmt.Sprintf(`from(bucket: "%s") `+
+			`|> range(start: %s, stop: %s) `+
+			`|> filter(fn: (r) => r["_measurement"] == "light_level_room" and r["_field"] == "level") `+
```

Comment:
Are we sure these queries are equivalent? I'm specifically looking at the `GROUP BY *` in the InfluxQL query, with no parallel here in the Flux. I think `group()` is the equivalent I'd expect here.

Reply:
I experimented with having `group()` in the flux query and/or not having the `GROUP BY *` in the InfluxQL query and it didn't seem to make much difference in the output & the results seemed equivalent - however! - I do see that the size of the returned dataset is slightly different if the "group" is there or not. I'll investigate more and do a follow-up.